src/rebar_prv_xref.erl

```diff
@@ -215,11 +214,15 @@ display_xref_result_fun(Type) ->
             {Source, SMFA, TMFA} =
                 case XrefResult of
                     {MFASource, MFATarget} ->
-                        {format_mfa_source(MFASource),
+                        Cwd = rebar_dir:get_cwd(),
+                        FormattedSource = format_mfa_source(MFASource),
+                        {rebar_dir:make_relative_path(FormattedSource, Cwd),
                          format_mfa(MFASource),
                          format_mfa(MFATarget)};
                     MFATarget ->
-                        {format_mfa_source(MFATarget),
+                        Cwd = rebar_dir:get_cwd(),
+                        FormattedTarget = format_mfa_source(MFATarget),
+                        {rebar_dir:make_relative_path(FormattedTarget, Cwd),
```

Comment:
Ideally, make the path relative to the directory returned by `rebar_dir:root_dir(State)`, which will let us not rely on the CWD location but rather a project-relative location. If the paths returned start with `_build/`, then we need to transpose the files analyzed and make them relative to the rebar_dir:lib_dir with the app name so it maps better to operating on source files (but iirc xref works directly on source so that should be fine)